posts/2024/2024_06_07_Inigo_week_1.rst

```diff
@@ -0,0 +1,30 @@
+First Week into GSoC 2024: Building the AutoEncoder, preliminary results
+========================================================================
+
+.. post:: June 7 2024
+    :author: Iñigo Tellaetxe
+    :tags: google
+    :category: gsoc
+
+
+What I did this week
+~~~~~~~~~~~~~~~~~~~~
+On my last post, I wrote about translating the AutoEncoder architecture from PyTorch to TensorFlow. I have successfully done so, and I have attempted to overfit the model with data from the `FiberCup dataset <https://tractometer.org/fibercup/home/>`__ to check whether the architecture is working as intended, and that the training objective is correctly defined (i.e. the loss function is correctly implemented).
```

Comment:
The rst URL syntax got an issue. 
Fix:
```
`separate repo <https://github.com/itellaetxe/tractoencoder_gsoc>`__
```

![image](https://github.com/dipy/dipy.org/assets/115863770/895fea21-a0d5-445a-b732-5c0aab2b1e3f)

Reply:
Fixed, thanks